tpm/Cargo.toml

```diff
@@ -6,9 +6,10 @@ name = "tpm"
 version = "0.1.0"
 
 [dependencies]
-anyhow = "1.0.81"
```

Comment:
Interesting, this version is different from all other versions. Please add a prerequisite patch before this consolidation patch to rectify that.

Reply:
Opened a PR for that here https://github.com/cloud-hypervisor/cloud-hypervisor/pull/7208